rootfs/Dockerfile

```diff
@@ -7,8 +7,11 @@ RUN adduser --system \
 	--group \
 	git
 
-RUN apt-get update \
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E1DF1F24 \
+	&& echo "deb http://ppa.launchpad.net/git-core/ppa/ubuntu xenial main" >> /etc/apt/sources.list \
+	&& apt-get update \
 	&& apt-get install -y \
+		git \
```

Comment:
We can pin the version, but apt repositories rarely store older versions of packages, so we would be forced to upgrade anyway